hackathon_site/event/models.py

```diff
@@ -44,3 +44,12 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.id} | {self.user.first_name} {self.user.last_name}"
+
+
+class UserActivity(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    sign_in = models.DateTimeField(null=True)
+    lunch1 = models.DateTimeField(null=True)
```

Comment:
Yeah, I'll think about it. My main idea was because we will have a known finite number of events so in the future we can set it up just like how we set up the application model questions, but if you want to extend it to further uses I can change it.